src/Symfony/Bundle/FrameworkBundle/Test/BrowserKitAssertionsTrait.php

```diff
@@ -124,6 +126,22 @@ public static function assertRouteSame($expectedRoute, array $parameters = [], s
         self::assertThat(self::getRequest(), $constraint, $message);
     }
 
+    public static function assertThatForResponse(Response $response, Constraint $constraint, string $message = ''): void
```

Comment:
no need for the first argument:
`$response = self::getResponse();`

Reply:
Done.